.github/workflows/ui-tests.yml

```diff
@@ -148,7 +148,7 @@ jobs:
         uses: actions/upload-artifact@v4
         if: ${{  failure() || cancelled() }}
         with:
-          name: raw-uitest-output-asan
```

Comment:
This one I guess was copypasta'd from the actual asan job further up in the file.